detectable/src/main/java/com/synopsys/integration/detectable/detectable/util/DependencyTypeFilter.java

```diff
@@ -0,0 +1,30 @@
+package com.synopsys.integration.detectable.detectable.util;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.function.Consumer;
+
+import org.jetbrains.annotations.Nullable;
+
+public class DependencyTypeFilter<T extends Enum<T>> {
```

Comment:
I found using the `ExcludeIncludeEnumFilter` here to be complicated and unnecessary.
I originally had issues with it being in an inaccessible sub-project, but was able to move it.
It added a lot of Configuration related classes into the DependencyTypeFilter and we aren't using the excluded list.